eventmesh-connectors/eventmesh-connector-jdbc/src/main/java/org/apache/eventmesh/connector/jdbc/connection/JdbcConnection.java

```diff
@@ -76,6 +73,29 @@ public JdbcConnection(JdbcConfig jdbcConfig, InitialOperation initialOperation,
         }
     }
 
+    /**
+     * Creates a ConnectionFactory that uses a pattern-based URL with placeholder values.
+     *
+     * @param urlWithPlaceholder The URL pattern with placeholders.
+     * @param replaces           The replacement values for the placeholders.
+     * @return The ConnectionFactory instance.
+     */
+    @SuppressWarnings("unchecked")
+    public static ConnectionFactory createPatternConnectionFactory(String urlWithPlaceholder, String... replaces) {
+        return config -> {
```

Comment:
This static method is called by another class, so you shouldn't move its position and it is recommended to leave it where it was at the bottom of this class.

There are other similar places, such as private enums in `MysqlCdcEngine` and those changes should be reverted.